<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>precipitation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>hours</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>hw (m/h)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_hw">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_period">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_precipitation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>mm/h</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_open_file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_file_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> File: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Insert</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>311</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="layout_histogram"/>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_open_file</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>read_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_precipitation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_time_3</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_period</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_time_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_precipitation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>rain_infiltration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_period</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>rain_infiltration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_period</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>change_period()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>read_file()</slot>
  <slot>rain_infiltration()</slot>
  <slot>change_period()</slot>
  <slot>change_precipitation()</slot>
 </slots>
</ui>
